lib/encoder_unittest.py

```diff
@@ -412,6 +413,15 @@ def testInit(self):
     cache = encoder.EncodingDiskCache(StorageOnlyContext())
     self.assertTrue(cache)
 
+  def testSearchPath(self):
+    # The score path starts out empty from setUp.
+    cache = encoder.EncodingDiskCache(StorageOnlyContext())
+    path = cache.SearchPath()
+    self.assertEquals(1, len(path))
```

Comment:
Because the path is the workdir + whatever's configured in scorepath. Added some msg=.